testing_and_setup/compass/ocean/scripts/plots/plot_initial_state.py

```diff
@@ -3,13 +3,14 @@
 This script creates plots of the initial condition.
 """
 # import modules
-import matplotlib.pyplot as plt
 from netCDF4 import Dataset
 import numpy as np
 import argparse
 import datetime
 import matplotlib
 matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from matplotlib.font_manager import FontProperties
```

Comment:
Usually, you need to switch to the `Agg` backend before importing `plt`.  I guess that isn't required in the most recent `matplotlib` but it's still safer to do it in this order.  PEP8 will complain about imports not being at the top but it is necessary in this case.